src/sentry/tasks/backfill_seer_grouping_records.py

```diff
@@ -79,6 +90,136 @@ def backfill_seer_grouping_records(
     Pass in last_processed_index = None if calling for the first time. This function will spawn
     child tasks that will pass the last_processed_index
     """
+
+    try:
+        project, redis_client, last_processed_index = initialize_backfill(
+            project_id, last_processed_index, dry_run
+        )
+    except FeatureError:
+        logger.info(
+            "backfill_seer_grouping_records.no_feature",
+            extra={"project_id": project_id},
+        )
```

Comment:
I think we can remove this log statement since it's also in `initialize_backfill`

Reply:
thanks! fixed (removed log in function)